MAINTAINERS.md

```diff
@@ -0,0 +1,66 @@
+# SPIRE Helm Charts Maintainership Guidelines and Processes
+
+This document captures the values, guidelines, and processes that the SPIRE Helm Charts project and its maintainers adhere to. All maintainers agree to uphold and abide by the text contained herein.
+
+For a list of active SPIRE Helm Charts maintainers, please see the [CODEOWNERS](CODEOWNERS) file.
+
+## General governance
+
+The SPIRE Helm CHarts project abides by the same [governance procedures][1] as the SPIFFE project, and ultimately reports to the [SPIFFE Steering Committee][3] (SSC) the same way that the SPIFFE project and associated maintainers do.
+
+SSC members do not track day-to-day activity in the SPIFFE/SPIRE projects, and this should be considered when deciding to raise issues to them. While the SSC has the ultimate say, in practice they are only engaged upon serious maintainer disagreement.
+
+## Maintainer responsibility
+
+SPIRE maintainers adhere to the [requirements and responsibilities][2] set forth in the SPIFFE governance document. They further pledge the following:
+
+* To act in the best interest of the project at all times.
+* To ensure that project development and direction is a function of community needs.
+* To never take any action while hesitant that it is the right action to take.
+* To fulfill the responsibilities outlined in this document and its dependents.
+
+## Guiding principles
+
+The goal of this project is to deliver Helm Charts to the user that are stable, secure, and simple.
+
+1. **Stability** -- The focus of this project is on stability over rapid feature development. SPIRE is a key infrastructure component that needs to be stable. While it is important to develop new features regularly, maintainers must take care that the project is stable.
+1. **Security** -- SPIRE is a security solution. Maintainers will ensure that the helm charts are secure along with guidance on increasing security for production environments.
+1. **Simplicity** -- Minimizing the options in the `values.yaml` files. Too many options can be confusing for the end user and we should limit options and to only expose configurations that are necessary. Its not possible to support every use case so we will select the use cases that are most important to the community.
```

Comment:
There will be few we can't support, although some might just require a bit more thinking and patience to come up with the most simple solution. In the end every usecase is possible if our consumers bring in things like kustomize to add the things we don't support via values files.

Reply:
I am not sure the goal should be minimizing the number of parameters in `values.yaml` file. Instead, I would suggest on focusing on producing well-behaving and well understood, intuitive default values, and provide clear and sufficient documentation on what the parameters mean, and how to use them, so users can deploy it "as is" with stable, expected, and consistent results.